.github/workflows/test.yml

```diff
@@ -1,18 +1,18 @@
 name: Tests
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
```

Comment:
Why `workflow_dispatch` is needed?

Reply:
to be able to trigger actions manually in GitHub

Reply:
I know, but why?

Reply:
it is handy sometimes

Reply:
This workflow does not depend on asynchronous/external resources. And if it fails, it will always fail. And if it is an intermitted failure (e.g. dead locks/out of memory/concurrency), The test or the code must be fixed anyway.

Given that, and the limited amount of time to run jobs in GitHub, I recommend we don't enable workflow_dispatch, so we force developers to understand the problem before starting over the tests.

Reply:
removed